core/ui/src/androidMain/kotlin/io/github/droidkaigi/confsched2023/ui/UserMessage.kt

```diff
@@ -99,7 +113,11 @@ class UserMessageStateHolderImpl : UserMessageStateHolder {
 interface UserMessageStateHolder {
     val messageUiState: MessageUiState
     fun messageShown(messageId: Long, userMessageResult: UserMessageResult)
-    suspend fun showMessage(message: String, actionLabel: String? = null): UserMessageResult
+    suspend fun showMessage(
+        message: String,
+        actionLabel: String? = null,
+        duration: SnackbarDuration? = null,
```

Comment:
I think the default parameter for the snackbar can be Short since the official implementation is doing so.
`duration: SnackbarDuration = SnackbarDuration.Short`

Reply:
As you can see below, the `duration` of `showSnackbar` in `Material3` seems to change its default value depending on whether there is an actionLabel or not, so I thought it would be better not to define it, but it would be better to set `SnackbarDuration.Short` ?